app/views/admin/booths/edit.html.haml

```diff
@@ -0,0 +1,3 @@
+%h1 Editing booth
```

Comment:
Let's add the booth's name here as well, so that we know which booth we are editing